notes/transcript15.md

```diff
@@ -213,108 +213,228 @@ Igor: I guess the signature won't signatures won't match but we can do it a shot
+
+### 1:05:00 - 1:10:00
+
+Oskar: whether it's sort of the timeframe recoverability or if we do a breaking change then should we have a path to recoverability, in some way we can limit the scope by creative solutions and so on. I think there is different sort of there's multiple options here that we can explore but it probably requires asynchronous effort.
+
+Ricardo: Yeah I agree with you. I just wanted to know who we should talk with to discuss this later on.
+
+Oskar: So I imagine Igor might want to do a few more runs on the post-mortem or whatever you want to call it pre-mortem, I don't know what you gonna call it, but then we can bring it up next core dev call and then we might we can also bring it up in a protocol call if that's interesting but I think.
+
+Ricardo: So basically use the Disqus post in this meeting.
+
 Igor: yeah I think Disqus is a good start and then we'll figure out from there. Because it's already a discussion started so there is no point to switch it somewhere else.
+
 Oskar: All right. Cool. So we have maybe 20 more minutes So Ricardo, you had a brief announcement on topic democracy.
+
 Ricardo: So tomorrow there is a meeting in the Status agenda that I'm going to present the swarm, what is the idea, that is being to the develop it in top democracy, basically how it works and why it works like this and so if you are interested in democracy and centralize the autonomous organizations and this can be used in many creative ways once it's created so might be interesting to many people.
+
 Oskar: Cool. Pedro you want to talk briefly about exposing cells goers in next package.
+
 Pedro: Yeah. So I think this is the the next logical step in terms of having reproducible builds. Right now we we host status code artifacts in digitalocean. So it's kind of a hand roll solution which really isn't required now that we have we hosted in this in this mix cache so we could just make status go another mix package and ideally also make the same the same approach that we took for status react does the same thing on status go so that people can just still do mix it up and mix takes care of all the dependencies and then that produces a next package which is uploaded to the next cache that we host and then that's what we use in Status react. So no more downloads during the the builds that are handled by us. It's a matter of consistency with all the other packages. I don't know if there's anything anything against this or if it's like logical for everyone. I don't know. It might be like for people who are working barely on Status Go that's maybe the people who will be more impacted by this. So it would just require that you do make shell and ensure that you're inside of this special shell from mix to build your app but it's not a big deal. You know from the experience with Status React, it's not a deal breaker. But I would be interested in hearing if there is anything against going forward with this.
```

Comment:
```suggestion
Pedro: Yeah. So I think this is the the next logical step in terms of having reproducible builds. Right now we host status-go artifacts in DigitalOcean. So it's kind of a home-grown solution which really isn't required now that we host this Nix cache, so we could just make status-go another Nix package and ideally also make the same approach that we took for status-react and do the same thing on status-go so that people can just still mix it up and Nix takes care of all the dependencies and then that produces a Nix package which is uploaded to the Nix cache that we host and then that's what we use in status-react. So no more downloads during builds that are handled by us. It's a matter of consistency with all the other packages. I don't know if there's anything against this or if it's like logical for everyone. People in status-go daily will be the people who will be most impacted by this. So it would just require that you do `make shell` and ensure that you're inside of this special shell from Nix to build your app, but it's not a big deal. We know from the experience with status-react that it's not a deal breaker. But I would be interested in hearing if there is anything against going forward with this.
```